status/status_test.go

```diff
@@ -119,6 +119,24 @@ func TestFromErrorOK(t *testing.T) {
 	}
 }
 
+func TestConvertKnownError(t *testing.T) {
+	code, message := codes.Internal, "test description"
+	err := Error(code, message)
+	s := Convert(err)
+	if s.Code() != code || s.Message() != message || s.Err() == nil {
```

Comment:
Checking `s.Err()` isn't necessary here; if there is a non-OK code, other tests should ensure that a status converts to a non-nil error.

Reply:
👍